myJS2.js

```diff
@@ -1,29 +1,29 @@
-
-function addTwoNumbers () {
-        $('#answer').text ( 
-            addNumbers( $('#num1').val(), $('#num2').val() )
-        );
+function addTwoNumbers() {
+    $('#answer').text(
+        addNumbers($('#num1').val(), $('#num2').val())
+    );
 }
 
 function multiplyNumbers() {
-     $('#answer').text (  
-        multiply( $('#num1').val(), $('#num2').val()  ) 
-     );
+    $('#answer').text(
+        multiply($('#num1').val(), $('#num2').val())
+    );
 }
 
 
-$(function() {
-  showBanner();
-  $('#btnAdd').on('click', addTwoNumbers);
+$(function () {
+    showBanner();
+    $('#btnAdd').on('click', addTwoNumbers);
 
 });
 
 
+//Changed from .show() to .toggle() so that the same button can be used to show and hide
 
-function showBanner(){
-    var hiddenBox = $("#banner-message" );
-    $("#button-container button" ).on( "click", function( event ) {
-        hiddenBox.show(2000);
+function showBanner() {
+    var hiddenBox = $("#banner-message");
+    $("#button-container button").on("click", function (event) {
+        hiddenBox.toggle();
```

Comment:
cool, good use of toggle, didn't strike me, I was thinking of creating another function to hide back :)